### ISSUE TITLE
how to set the alternate color to my report data

### ISSUE THREAD

**Issue by akshayauti** [optimistex/xlsx-template-ex#18]
I want to set alternate colors to row in sheet and I am not able to do that using alternate color theme in template, the generated report ignore the colors

**Comment by optimistex** [comment]
@akshayauti Yes, it does not care about the colors yet. Unfortunately, I do not have enough time to add this. If you added a pull request, I'd review and release it.
